### ISSUE TITLE
Discuss the format...

### ISSUE THREAD

**Issue by rogerhoward** [codeandcoffeelb/hacksagainsthumanity#2]
Is this a weekend-long event? A one-day event? Thoughts on format, overall?

**Comment by rogerhoward** [comment]
1. Awards - I think a major goal should be an award to everyone who participates. The sillier, the better. Perhaps a few "real" awards with big prizes? But for the most part, this should be Razzberry Awards-level fun.

**Comment by MJLang** [comment]
Yeah, not a big fan of putting money in hackathons. Especially if its "for the lulz". 
Should be more of a "I was here" price instead of a "look how much better I am".

**Comment by rogerhoward** [comment]
:+1: 

Ok, so status-only prizes. All for fun. Self deprecation required.

**Comment by davidnuon** [comment]
How do we feel about self-deprecating certificates? 

![](https://cloud.githubusercontent.com/assets/477742/12988158/50235dfe-d0b4-11e5-811a-e6713f153587.png)
